### ISSUE TITLE
Game Not Loading

### ISSUE THREAD

**Issue by Blueturbo47** [awWhy/Fundamental#3]
Was about almost finished with the accretion stage, the one with the dust and mass. And now game is no longer loading from that save anymore.

**Comment by awWhy** [comment]
Better to show console for errors next time
But that issue was most likely caused with Screen reader support missing related ID (only if SR support was ON)
Was fixed few minutes ago
